x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -212,14 +213,15 @@ export function TabContent({
   currentTab
 }: {
   error: APMError;
-  currentTab?: string;
+  currentTab?: { key: string; label: string };
```

Comment:
I see this quite a few places. Can you make it an explicit interface in `PropertiesTable/index.ts`:

```ts
export interface Tab { 
  key: string; 
  label: string 
}
```

Thanks!